components/automate-ui/src/app/pages/client-runs/client-runs.component.ts

```diff
@@ -312,12 +312,14 @@ export class ClientRunsComponent implements OnInit, OnDestroy {
       // We want to report total nodes to telemetry, but only when there are no
     // filters. This way we can see how many nodes a customer has connected to
     // automate in total.
-    combineLatest(
+    combineLatest([
       this.nodeCounts$.pipe(distinctUntilKeyChanged('total')),
       this.numberOfSearchBarFilters$.pipe(distinctUntilChanged())
-    )
+    ])
     .pipe(takeUntil(this.isDestroyed))
-    .subscribe(([nodeCounts, filterCount]) => {
+    .subscribe((values: any[]) => {
+      const nodeCounts: NodeCount = values[0];
+      const filterCount: number = values[1];
```

Comment:
This is a slightly peculiar instance of the `combineLatest` change. This is the first occurrence that chains it to `subscribe` instead of `map`. But `subscribe` did not take kindly when I tried the usual fix, which would have been to add types to the destructured elements:
```
subscribe(([nodeCounts, filterCount]: [NodeCount, number])
```
So I found the above workaround was necessary.

Reply:
> But subscribe did not take kindly when I tried the usual fix [...]

What happened? Did it throw you out? 😉